lib/charms/layer/apache_bigtop_base.py

```diff
@@ -491,6 +491,31 @@ def get_hadoop_version():
     return parts[1]
 
 
+def get_package_version(pkg=None):
```

Comment:
What is the reason for accepting None here? Wouldn't it be simpler to just force the user to pass in a package?